values.js

```diff
@@ -1,25 +1,155 @@
 const values = {
-  'absolute': 'absolut',
+  // global values
+  'inherit': 'warisan',
+  'initial': 'awal',
+  'revert': 'balikkan',
+  'revert-layer': 'balikkan-lapisan',
```

Comment:
same as above "kembalikan-lapisan" is better I think

Reply:
I agree. I'll change